pkg/transport/test/transformer.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/stretchr/testify/require"
```

Comment:
This file should be moved back to binding/test

Reply:
moved back.